cli/tests/integrations/test_package.py

```diff
@@ -667,11 +670,37 @@ def test_list_cli():
 
     stdout = file_json(
         'tests/data/package/json/test_list_helloworld_cli.json')
-    _list(stdout=stdout)
+    _list(args=['--json'], stdout=stdout)
 
     _uninstall_cli_helloworld()
 
 
+def test_list_cli_flag():
+    kafka_path = 'tests/data/package/json/test_list_kafka_entry.json'
+    kafka_json = file_json_ast(kafka_path)
+    helloworld_path = 'tests/data/package/json/test_list_helloworld_entry.json'
```

Comment:
lets just take the first element from`cli/tests/data/package/json/test_list_helloworld_cli.json`